src/BasicInputs/components/NumberFormatCustom.js

```diff
@@ -10,7 +10,7 @@ function NumberFormatCustom (props) {
 
   const valuesRange = (inputObj) => {
     const { value } = inputObj;
-    if (min <= value && value <= max) return inputObj;
+    if (min <= value && value <= max || min < 0 && value === '-') return inputObj;
```

Comment:
Could you please add parenthesis, so the operators priority is clear?
I assume you meant `(min <= value && value <= max) || (min < 0 && value === '-')`
Nontheless, `(min < 0 && value === '-')` doesn't seem right: indeed, '-' is not a number, and therefore can't be compared to a number.
I'd suggest the following:
`value === '-' || (min <= value && value <= max)`

Reply:
I fixed the comparisons order and placed parenthesis correctly

Reply:
Thanks!